proxy/coco/src/lib.rs

```diff
@@ -25,7 +25,7 @@
 
 pub use librad::{
     git::{self, identities::local::LocalIdentity, include, local::url::LocalUrl, Urn},
-    identities::{Person, Project},
+    identities::{payload as identities_payload, Person, Project},
```

Comment:
I think `self` would be "nicer" :) Makes me think why not just re-export `librad` though really :thinking:

Reply:
Fixed:
```
identities::{
        payload::{self as identities_payload},
        Person, Project,
    },
```
Is this what you meant? IMO the old one was a little nicer, but I'm fine with this change :)

Reply:
Whoops, should have been clearer. I was saying that we should just re-export `identities`, and as @NunoAlexandre commented somewhere else, we would use it as `coco::identities::payload` :)

Reply:
Fixed. I hope that we haven't used up a precious name :grimacing: